### PR TITLE
Migrated to Minecraft version 1.20.5

### PR DESCRIPTION
**Pull request by TeddyBear2004**
Currently the IntelliJ Configuration 'Minecraft Client' does not work on the branch, because 'fabric-screen-api-v1 2.0.15+6f7ba8f23b' contains a mixin that is faulty due to the update. However, one built by 'gradlew build' works.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -3,14 +3,14 @@ mod_id = authme
 mod_name = Auth Me
 mod_description = Authenticate yourself in Minecraft and re-validate your session
 ## {x-release-please-start-version}
-mod_version = 8.0.0+1.20.4
+mod_version = 8.0.0+1.20.5
```
Comment (axieum) [inline gradle.properties:6]:
```suggestion
mod_version = 8.0.0+1.20.4
```
We'll let the CI pipeline update the version. :)

File: gradle.properties [hunk 2]
```diff
@@ -3,14 +3,14 @@ mod_id = authme
...
 ## {x-release-please-end}
 
 # Fabric
-minecraft_version = 1.20.4
-loader_version = 0.15.1
-yarn_mappings = 1.20.4+build.1
-fabric_version = 0.91.2+1.20.4
+minecraft_version = 1.20.5
+loader_version = 0.15.10
+yarn_mappings=1.20.5+build.1
```
Comment (axieum) [inline gradle.properties:12]:
```suggestion
yarn_mappings = 1.20.5+build.1
```

File: gradle.properties [hunk 3]
```diff
@@ -31,7 +31,7 @@ cf_relations_incompatible =
 
 # Modrinth
 mr_project_id = yjgIrBjZ
-mr_game_versions = 1.20.3, 1.20.4
+mr_game_versions = 1.20.3, 1.20.4, 1.20.5
```
Comment (axieum) [inline gradle.properties:34]:
```suggestion
mr_game_versions = 1.20.5
```

File: build.gradle [hunk 4]
```diff
@@ -17,7 +17,7 @@ allprojects {
     version = project.mod_version
     archivesBaseName = project.mod_id
 
-    sourceCompatibility = targetCompatibility = JavaVersion.VERSION_17
+    sourceCompatibility = targetCompatibility = JavaVersion.VERSION_21
```
Comment (axieum) [inline build.gradle:20]: Let's update the CI/CD pipeline to use Java 21 too please.
https://github.com/axieum/authme/blob/1ad64813853d0badce50215ec400ed65734e8450/.github/workflows/build.yml#L26-L30
https://github.com/axieum/authme/blob/1ad64813853d0badce50215ec400ed65734e8450/.github/workflows/release.yml#L47-L51

**Review by axieum — CHANGES_REQUESTED** [review]
Love this, thank you! Just a few minor tweaks if you have time. :)

**Comment by axieum** [comment]
> Here are the changes, But I still need to work on the GUI: image

Oh wow, yeah okay, that is interesting. I suppose we need to render the text after the new blur is applied hahahaha.

**Comment by TeddyBear2004** [comment]
Got it up and running.

**Comment by axieum** [comment]
Thanks. I haven't forgotten about this one, just getting up and running on my local soon.

**Review by axieum — APPROVED** [review]

**Pull request merged**